src/styles/style.css

```diff
@@ -74,36 +73,36 @@
   color: var(--lightBlue);
 }
 
-.darkGray {
+.card__darkGray {
   color: var(--darkGray);
 }
 
-.card__common-text--sizeUp {
+.card__commonText--sizeUp {
   font-size: 12px;
   line-height: 18px;
 }
 
 /************************position**************************/
 
-.card__div--center {
+.card__div {
   margin-top: 32px;
   margin-bottom: 40px;
   display: flex;
   justify-content: center;
 }
 
-.card__header--margin_bottom {
+.header-position {
   margin-bottom: 4px;
 }
 
-.card__common-text--margin_bottom {
+.commonText-position {
   margin-bottom: 16px;
 }
 
-.card__reviewes--margin_bottom {
+.card__reviewes--position {
   margin-bottom: 24px;
 }
 
-.card__price--margin_bottom {
+.card__price--position {
   margin-bottom: 16px;
 }
```

Comment:
You don't need separate classes for setting margin-bottom

`card__reviews` - sets all styles related to position of the element `reviews` inside block `card`. Margins goes here
`reviews` - sets innert styles for reviews element
so your element would have only two classes `<div class='card__reviews reviews'>`